.github/workflows/release.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: LuaRocks Upload
-        uses: nvim-neorocks/luarocks-tag-release@v1.0.2
+        uses: nvim-neorocks/luarocks-tag-release@v4.0.0
```

Comment:
Should be changed to version `v4` instead of `v4.0.0` so dependabot understands to only bump this on major version changes.